src/containers/Projects/ProjectCard/index.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {FC, useEffect, useState} from 'react';
+
+import {getUser} from 'apis/users';
+import {api as projectApi} from 'apis/projects';
+import {Avatar} from 'components';
+import Button from 'components/Button';
+import {useWindowDimensions} from 'hooks';
+import {User} from 'types/app/User';
+import {Icon, IconType} from '@thenewboston/ui';
+
+import './ProjectCard.scss';
+
+type Props = {
+  description: string;
+  id: string;
+  logoUrl: string;
+  projectLead: string;
+  title: string;
+};
+
+const ProjectCard: FC<Props> = ({description, id, logoUrl, projectLead, title}) => {
+  const [projectLeadUser, setProjectLeadUser] = useState<User | null>(null);
+  const {width} = useWindowDimensions();
+
+  useEffect(() => {
+    (async function () {
+      // We require to call two extra APIs just to get the project lead's name, perhaps we
+      // should adopt a BFF design
+      const projectMemberResponse = await projectApi.getProjectMemberById(projectLead);
```

Comment:
hi @angle943 @ThaDeveloper we are required to call two extra APIs on top of the `projects` api just to obtain the project lead's display name. Is it possible to nest the lead's `project member` as well as the `user` data within the `projects` api?

Reply:
This wouldn't  be a recommended API infrastructure. Imagine the load we be adding  to Projects. You only need `display name`?

Reply:
@ThaDeveloper it looks like we only need the `display name`

Reply:
yes only need `display name`